src/include/sof/schedule/schedule.h

```diff
@@ -30,6 +30,7 @@
 enum {
 	SOF_SCHEDULE_EDF = 0,	/* EDF scheduler */
 	SOF_SCHEDULE_LL_TIMER,	/* Low latency timer scheduler */
+	SOF_SCHEDULE_LL_DMA,	/* Low latency DMA scheduler */
```

Comment:
Can we add more time domain detail in the comments here for all scheduler types. e.g.  
```
/* Low latency DMA scheduler - schedules immediately on DAI DMA IRQs */
```

Reply:
Done.